### DISCUSSION TITLE
Get a load of this guy

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#46] [General (Memes)]
https://github.com/NeoStudios/Charles-EngineNEW

all they did was just change the name of the engine and claimed it as theirs lmao, sadly, i can't do anything about it

what makes this even funnier is the fact they didn't even try to change the assets for the title screen

**Comment by charlesisfeline** [comment]
WHO STOLE MY ENGINE

**Reply by ThatOneSillyGuyDon** [reply]
this clown i just put the link to lol

wow, not only did they take the code from my engine and claim it as theirs, but they stole your engine name too lmfao, that's how you know they're a little child who has no originality

**Reply by TrevorAnimates** [reply]
This was the guy who stole ur engine lol

**Reply by charlesisfeline** [reply]
yea

but how do we just take this down

**Reply by charlesisfeline** [reply]
plus, i made that engine and someone forked it and claimed it
fuc

**Reply by TrevorAnimates** [reply]
Wat no @SonicProYT ur the one u copied the code AHAHAHA

**Reply by charlesisfeline** [reply]
yea, but i removed the fork

but SOMEONE LITERALLY FORKED IT AFTER I TOOK IT DOWN

**Reply by Cloudiiie** [reply]
> 

dude you didn't even HELP make Demolition Engine dude, stop claiming it as yours

**Reply by charlesisfeline** [reply]
i might help only if i have a music player hx file

**Reply by ThatOneSillyGuyDon** [reply]
Lmao, all my homies looking at the wrong person, when they originally forked it to port a mod or something

**Comment by Cloudiiie** [comment]
lmao, dude tried to be sneaky

**Reply by Cloudiiie** [reply]
plus, it literally says "forked from DemolitionEngine"

**Reply by ThatOneSillyGuyDon** [reply]
and that's what makes him the entire circus

**Comment by Theoyeah** [comment]
LETS ATTACK THIS REPO
